crest/Assets/Crest/Crest/Shaders/OceanConstants.hlsl

```diff
@@ -13,6 +13,9 @@
 // C# code to check if any parameters are within the MAX_LOD_COUNT limits
 #define MAX_LOD_COUNT 15
 
+// The size of the largest LOD.
```

Comment:
is this the size of the largest lod? i think it might be good to expand this comment and rename the symbol to make it really explicit and clear. i cant quite remember the background for this, i guess its in our thread but it would be good to make the code really clear

Reply:
I have since removed it as the issue was actually a bug in C#.